packages/react/dataconnect-sdk/js/default-connector/index.cjs.js

```diff
@@ -0,0 +1,79 @@
+const { queryRef, executeQuery, mutationRef, executeMutation, validateArgs } = require('firebase/data-connect');
```

Comment:
on moving js-dc-sdk to workspace, noticed a cleanup was done under `/angular` via [f246a1e](https://github.com/invertase/tanstack-query-firebase/pull/166/commits/f246a1efe88a028c8b37f621a6adb079cdde1629), shouldn't the following be removed as well ?
- [/react/dataconnect](https://github.com/maneesht/tanstack-query-firebase/tree/main/packages/react/dataconnect) 
- [/react/dataconnect-sdk/js/default-connector](https://github.com/maneesht/tanstack-query-firebase/tree/main/packages/react/dataconnect-sdk/js/default-connector)